### PR TITLE
Added new module stanford_saml_block

### PR DESCRIPTION
**Pull request by sherakama**
# Ready.

_To test:_
1. Build a local JSA with the "anchorage" option
2. Check out this branch
3. Install `drush si stanford_sites_jumpstart_academic install_configure_form.itasks_extra_tasks=anchorage`
4. Navigate to /user
5. See form and test login.

Please provide feedback on choice of language as well as the usual code review.

### CODE REVIEW

**Comment by jbickar** [comment]
Step away from the computer, @sherakama.

**Comment by jbickar** [comment]
@sherakama refresh my memory: we are doing this in `stanford_jumpstart_login` currently?

**Comment by sherakama** [comment]
Yep: https://github.com/SU-SWS/stanford_jumpstart/blob/7.x-4.x-dev/modules/stanford_jumpstart_login/stanford_jumpstart_login.context.inc#L46

**Comment by jbickar** [comment]
Uh, shouldn't this be all lowercase? `stanford_saml_block`?

**Comment by sherakama** [comment]
Good point. Will fix.

**Comment by jbickar** [comment]
1. Please change the module name to `stanford_saml_block`
2. `README` is a mess
3. Our convention is that Context machine names use underscores rather than dashes; please change Context machine name to `stanford_simplesamlphp_login`
4. `stanford_saml_block.install` file is unnecessary

**Comment by jbickar** [comment]
It works, though :)

**Comment by sherakama** [comment]
Haha, thanks. The PR does say "not ready" but I welcome your feedback.

**Comment by jbickar** [comment]
_Y U ASSIGN TO ME IF NOT READY?_

**Comment by sherakama** [comment]
Because. Raisons.

**Comment by jbickar** [comment]
I figured since everything else in this PR was so slapdash that _not ready_ was just an oversight too :joy:

**Comment by sherakama** [comment]
Should I add HSDO-32 to this?
https://stanfordits.atlassian.net/browse/HSDO-32

**Comment by jbickar** [comment]
Seems like a logical place.

File: modules/stanford_saml_block/stanford_saml_block.info [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name = Stanford SAML Login Block
+description = Places the SUNetID Login block
+core = 7.x
+package = Anchorage
+version = 7.x-1.x
+dependencies[] = block
+dependencies[] = context
+dependencies[] = ctools
+dependencies[] = features
+features[context][] = stanford_simplesamlphp_login
```
Comment (jbickar) [inline modules/stanford_saml_block/stanford_saml_block.info:11]: Do we want a dependency on `stanford_ssp` here?

File: modules/stanford_saml_block/stanford_saml_block.module [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  $form['saml_auth']['saml_link'] = array(
+    '#prefix' => "<p>",
+    '#markup' => l(t("Log in with your SUNet ID »"), "saml_login"),
+    '#suffix' => "</p>",
+  );
+
+  $form['drupal_auth'] = array(
+    '#type' => 'fieldset',
+    '#title' => "Drupal Login",
```
Comment (jbickar) [inline modules/stanford_saml_block/stanford_saml_block.module:32]: Do we want this to be collapsible and collapsed, as we do with WMD?

Comment (sherakama) [inline modules/stanford_saml_block/stanford_saml_block.module:32]: I never liked that the option was hidden. Messed with Behat tests. I was more going for visual grouping. I didn't even nest the actual form items.

File: modules/stanford_saml_block/stanford_saml_block.info (hunk 1 above)
Comment (sherakama) [inline modules/stanford_saml_block/stanford_saml_block.info:11]: I guess so.

File: modules/stanford_saml_block/stanford_saml_block.module (hunk 2 above)
Comment (jbickar) [inline modules/stanford_saml_block/stanford_saml_block.module:32]: Sure. Let's let the UX person fight that battle.

Comment (jbickar) [inline modules/stanford_saml_block/stanford_saml_block.module:32]: Ehrmagerd. Github has emoji reactions now.

**Comment by sherakama** [comment]
Thanks!

**Pull request merged**